plutus-core/untyped-plutus-core/test/Evaluation/Machines/Budget/Examples/InterList/FoldrInterList.plc.golden

```diff
@@ -3,9 +3,9 @@
 | mem: 0
 })
 | BAST causes ({ cpu: 0
-| mem: 46
+| mem: 116
```

Comment:
And in other places. Really don't get it.

Reply:
I've figured it out: we now have a general instance `ExMemoryUsage` instance for `[a]` and we define the `ExMemoryUsage Text` in terms of `String`, which has changed and we use `Text` in the names of variables, hence the change.
@kwxm do we care?

Reply:
> @kwxm do we care?

I don't think so.  We may have to have a charge for the overall size of the AST, but it doesn't particularly matter if the size changes a bit (and maybe we could base such a charge on the serialised size, instead of doing a potentially expensive traversal of the deserialised AST).  The stuff we're using on the chain shouldn't contain any variable names anyway, and we're probably not going to be using strings much.  I've got rid of `BAST` in my recent PR and replaced it with something called `BStartup` which is supposed to account for startup costs, but I don't have enough data to work out what that should be in general and it's currently just a constant (but that may change later).

Reply:
Thanks, I also thought it probably wouldn't matter.